searx/webapp.py

```diff
@@ -761,6 +761,11 @@ def search():
         )
     )
 
+    # timings: get engine response times sorted from slowest to fastest
+    timings = webutils.get_engine_timings(result_container.get_timings())
+    # first entry in list is the slowest engine. [0] on object is engine name, [1] is total time
+    max_response_time = timings[0][1] if len(timings) > 0 else None
```

Comment:
```
timings[0][1] if len(timings) > 0 else None
```
could be simplified to
```
timings[0][1] if timings else None
```

Reply:
Didn't know an empty list returned false, the more you know.